frontend/src/Wanted/CutoffUnmet/CutoffUnmet.js

```diff
@@ -252,7 +252,7 @@ class CutoffUnmet extends Component {
                         Are you sure you want to search for all {totalRecords} Cutoff Unmet episodes?
                       </div>
                       <div>
-                        This cannot be cancelled once started without restarting Sonarr.
+                        This cannot be cancelled once started without disabling all of your indexers.
```

Comment:
Restarting Sonarr is still a valid way to cancel, why not list both?

Reply:
Is it? As far as I'm aware (and what everyone tells us in support channels), it continues on startup again and a restart is not a valid way to stop a search-all.

Reply:
I just tested it with Cutoff Unmet, and it does seem to interrupt the process. This is different behavior than the other arrs, so if you would like to decline this PR, I'm fine with that.

Reply:
That's news to me they don't behave the same way, in Sonarr any in progress task that is not completed when Sonarr starts is tossed out, scheduled tasks would be re-queued by the scheduler, user queued tasks are just lost. I'm curious why this was changed downstream, maybe @Qstick knows?

Reply:
I'm not sure it was intentional if it did change

Reply:
So I think this will only be different in Radarr and not the others. It's because we don't have a wanted page and we always use normal search commands when firing UI searches, never the singular cutoff unmet or missing search commands

Reply:
Ahh that makes sense, in which case, I think restarting Sonarr or disabling all indexers is what we want to say here.